packages/core/contracts/chainbridge/SinkGovernor.sol

```diff
@@ -0,0 +1,50 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.0;
+
+import "../oracle/interfaces/FinderInterface.sol";
+import "../oracle/implementation/Constants.sol";
+
+/**
+ * @title Governor contract on L2 that receives governance actions from L1.
+ */
+contract SinkGovernor {
```

Comment:
Thoughts on refactoring the logic in this contract into a shared lib between `oracle/Governor` and this contract? Just saying because it reuses logic from Governor.

Reply:
This is definitely non-blocking though, just a side comment

Reply:
I'll put together a change to do this, but I would prefer to do it in a follow-up just so OZ can review the change to the Governor independently and make sure it causes no functional changes.